vllm/engine/arg_utils.py

```diff
@@ -368,9 +368,12 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                             action='store_true',
                             help='Disables sliding window, '
                             'capping to sliding window size')
-        parser.add_argument('--use-v2-block-manager',
-                            action='store_true',
-                            help='Use BlockSpaceMangerV2.')
+        parser.add_argument(
+            '--use-v2-block-manager',
+            default=EngineArgs.use_v2_block_manager,
+            action='store_true',
```

Comment:
Don't you need to set this to `store_false`?

My understanding is that `store_true` set `use_v2_block_manager` to True if you specify `--use-v2-block-manager`, and otherwise set it to false by default.

Reply:
Resolving the comment since I think with this action if the user specifies --use-v2-block-manager then it will be set to true and if it is not specified then the default (EngineArgs.use_v2_block_manager) is used which is true. Let me know if it needs changes.

Reply:
Yep thanks!